repository-ydb-v2/src/test/java/tech/ydb/yoj/repository/ydb/merge/QueriesMergerTest.java

```diff
@@ -164,14 +161,6 @@ private <T extends Entity<T>> YdbRepository.Query<?> insert(T p) {
         return new YdbRepository.Query<>(new InsertYqlStatement<>(tableDescriptor, schema), p);
     }
 
-    @SuppressWarnings("unchecked")
```

Comment:
Just unused code (since initial YOJ commit to open source, actually :zany_face:)